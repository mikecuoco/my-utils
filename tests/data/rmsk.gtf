version https://git-lfs.github.com/spec/v1
oid sha256:cc779286f8af2989ca4c0d70a3c51e18cadaa685657052b73613e5b8c4fa7809
size 5557
