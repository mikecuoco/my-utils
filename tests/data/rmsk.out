version https://git-lfs.github.com/spec/v1
oid sha256:43d89a4878105bc3c7680ff82c29cf1f78cc7f082f65a6408abba714f94d03f2
size 1606793
